.github/workflows/release-please.yml

```
@@ -14,20 +14,20 @@ jobs:
     outputs:
       release_created: ${{ steps.release.outputs.release_created }}
     steps:
-      - uses: GoogleCloudPlatform/release-please-action@v3
+      - uses: googleapis/release-please-action@v4
```

```suggestion
      - uses: googleapis/release-please-action@v17
```

I guess there is quite a new version...

---

The latest I can see is `4.2.0`:
https://github.com/googleapis/release-please-action/releases

Am I missing something?